src/contracts.ts

```diff
@@ -30,16 +35,16 @@ export class Contracts {
     }
 
     async initialize(): Promise<void> {
```

Comment:
Maybe we don't need this function to be async anymore. Furthermore, we could move this code in the `constructor` or make it `private`

Reply:
This will fixed in the next commit.